src/view/reminder/reminder-time-options.ts

```diff
@@ -16,7 +16,7 @@ import { AlarmType } from '../../types/event';
 // 	id: string;
 // 	label: string;
 // 	icon?: string;
-// 	click?: (e: React.SyntheticEvent<HTMLElement> | KeyboardEvent) => void;
+// 	onClick?: (e: React.SyntheticEvent<HTMLElement> | KeyboardEvent) => void;
```

Comment:
Should these commented types be removed entirely?

Reply:
I think so, I will do it in a future task dedicated exclusively to clean up dead code from the project